terraform/monitoring/02_dashboards.tf

```diff
@@ -32,12 +32,32 @@ resource "google_monitoring_dashboard" "adservice_dashboard" {
   dashboard_json = file("./dashboards/adservice_dashboard.json")
 }
 
-# Ceates a dashboard for the recommendationservice. The details of the charts
+# Creates a dashboard for the recommendationservice. The details of the charts
 # in the dashboard can be found in the JSON specification file.
 resource "google_monitoring_dashboard" "recommendationservice_dashboard" {
   dashboard_json = file("./dashboards/recommendationservice_dashboard.json")
 }
 
+# Creates a dashboard for the cartservice.
+resource "google_monitoring_dashboard" "cartservice_dashboard" {
```

Comment:
Most of these files are a very similar.  Why aren't they template-tized?

Reply:
Hmm good point - they are very similar but there are some small differences between which charts are in which dashboards.

I guess it's possible to uniform the charts across 3 of the dashboards but I'm not too sure how worthwhile having two template examples would be. These charts also contain OpenCensus metrics and if there are some changes in metric titles when moving to OpenTelemetry it might be better to have the dashboards separated. Please let me know what you think!

Reply:
Unless I'm mistaken the main difference is in chart titles which include the service name.  Otherwise the metrics, alignments, alignment periods, etc. all look the same so why can't you just have a template that takes the service name?

Reply:
There are a different number of charts in each dashboard and different metrics - there are small differences between each dashboard. Ex: Cartservice tracks memory usage while currencyservice tracks CPU usage.

Reply:
Brian and I talked offline about this - since the dashboards are different we're gonna keep them as-is.

Reply:
Sounds good.